apps/retrospective/serializers/retrospective_feedback.go

```diff
@@ -0,0 +1,42 @@
+package serializers
+
+import (
+	"github.com/iReflect/reflect-app/apps/retrospective/models"
+	"github.com/iReflect/reflect-app/apps/user/serializers"
+	"time"
+)
+
+// RetrospectiveFeedback ...
+type RetrospectiveFeedback struct {
+	ID              uint
+	SubType         string
+	Type            models.RetrospectiveFeedbackType
+	RetrospectiveID uint
+	Text            string
+	Scope           models.RetrospectiveFeedbackScope
+	AssigneeID      *uint
+	Assignee        *serializers.User
+	AddedAt         *time.Time
+	ResolvedAt      *time.Time
+	ExpectedAt      *time.Time
+	CreatedByID     uint
+	CreatedBy       serializers.User
+}
+
+// RetrospectiveFeedbackUpdateSerializer ...
+type RetrospectiveFeedbackUpdateSerializer struct {
+	Text       *string    `json:"Text"`
+	Scope      *int8      `json:"Scope"`
+	AssigneeID *uint      `json:"AssigneeID"`
+	ExpectedAt *time.Time `json:"ExpectedAt"`
+}
+
+// RetrospectiveFeedbackCreateSerializer ...
+type RetrospectiveFeedbackCreateSerializer struct {
+	SubType string `json:"subType" binding:"required"`
```

Comment:
We need to add validations for enums

Reply:
Not needed for now as sub-type is not defined/hardcoded in string

Reply:
We will still need it for Scope

Reply:
done